app/json/slate_configs.json

```diff
@@ -778,8 +778,8 @@
   },
   {
     "id": "8a495385-9972-4d0e-b93e-4234f4897939",
-    "displayName": "Algorithmic Sports Slate",
-    "description": "Popular with Pocket readers  Stories from across the web",
+    "displayName": "Popular with Pocket readers  Stories from across the web",
```

Comment:
This seems like two sentences are joined together

Reply:
it's literally what is on the topic pages now.  In other words, we are managing it, but someone else created it.

Reply:
```
      <SectionHeader
        sectionTitle={t('discover:curated-by-our-editors', 'Curated by our editors')}
        sectionDescription={t('discover:stories-to-fuel-your-mind', 'Stories to fuel your mind')}
      />
```

It is hardcoded at the moment, but would be awesome if it came down so curation could tweak it as will.

Reply:
It sounds like we should to add a title property to slates.

We also didn't get around to localizing these strings. Maybe we can tackle both of those at once when we're doing another recommendation api project.

Reply:
Am I wrong it would still be worth making this switch in the short term @collectedmind ?  If so, I will update this PR with an additional small lineup change we discussed this morning @mmiermans and re-request review.